app/assets/javascripts/components/common/feedback.jsx

```diff
@@ -116,15 +116,15 @@ const Feedback = ({ assignment, username, current_user }) => {
       feedbackList.push(<li key={i.toString()}>{messages[i].message}</li>);
     }
 
-    if (!showFeedbackForm) {
-      feedbackButton = <a onClick={showFeedbackInput} className="button dark">{I18n.t('courses.suggestions_feedback')}</a>;
-    } else {
+    if (showFeedbackForm) {
       feedbackForm = (
         <form onSubmit={handleSubmit}>
           <textarea className="feedback-form" rows="1" cols="150" ref={inputRef} placeholder={I18n.t('courses.suggestions_feedback')} />
           {submitFeedback}
         </form>
       );
+    } else {
+      feedbackButton = <a onClick={showFeedbackInput} className="button dark">{I18n.t('courses.suggestions_feedback')}</a>;
```

Comment:
This is a condition that rendered the  button for displaying the text_area for submitting a feedback to the course, initially the condition was not properly structured, hence the button was not displaying and since it was related to the component i was fixing I just added the fix